src/ui/library/BottomSheet.tsx

```diff
@@ -43,10 +53,10 @@ export function useBottomSheet() {
         contentHeight={animatedContentHeight}
         enablePanDownToClose={true}
         animateOnMount={true}>
-        <BottomSheetView onLayout={handleContentLayout}>{children}</BottomSheetView>
+        <BottomSheetView onLayout={handleContentLayout}>{children ? children : noContent}</BottomSheetView>
```

Comment:
We need this because passing `undefined` or `null` as children to the BottomSheet makes it behave unexpectedly. This is just to handle and edge case in the [dynamic content generation for BottomSheet](https://github.com/litentry/litentry-app/blob/effddf0dfc3146afca243f9388b254864bf0d5e3/src/ui/screens/AccountsScreen.tsx#L66), but ideally the BottomSheet will never be opened without any content.

Reply:
An updated implementation of [dynamic BottomSheet](https://github.com/litentry/litentry-app/blob/effddf0dfc3146afca243f9388b254864bf0d5e3/src/ui/screens/AccountsScreen.tsx#L66) is [here](https://github.com/litentry/litentry-app/pull/1179)

But we still need to check if the children is null.